keyboards/westfoxtrot/cypher/rev5/info.json

```diff
@@ -0,0 +1,207 @@
+{
+    "keyboard_name": "Cypher",
+    "width": 19.5,
+    "height": 5,
+    "layouts": {
+      "LAYOUT_ansi": {
```

Comment:
This file doesn't match the `LAYOUT_all` macro in `rev5.h`, but if you'd prefer, I could probably fix it in another PR.